.changeset/loud-readers-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": minor
```

Comment:
this would be a patch if we did it (changing an existing api in a non-breaking way)